optuna/samplers/_tpe/sampler.py

```diff
@@ -147,10 +149,74 @@ def objective(trial):
                 Added in v2.2.0 as an experimental feature. The interface may change in newer
                 versions without prior notice. See
                 https://github.com/optuna/optuna/releases/tag/v2.2.0.
+        group:
+            If this and ``multivariate`` are :obj:`True`, the multivariate TPE with the group
+            decomposed search space is used when suggesting parameters.
+            The sampling algorithm decomposes the search space based on past trials and samples
+            from the joint distribution in each decomposed subspace.
+            The decomposed subspaces are a partition of the whole search space. Each subspace is a
+            maximal subset of the entire search space under the condition that they are disjoint
+            from each other and that a trial exists whose parameter set includes the subspace.
```

Comment:
Maybe it is incorrect. For example,

- search spaces are {x, y} and {y, z}
- subspaces are {x, y} and {z}

This case satisfies the description, but subspaces should be {x}, {y}, and {z}.

Reply:
:tashikani: Thank you for the comment. I updated the description.

Reply:
Could you remain the word "maximal"? The following case satisfies the definition, but the subspace should be {x, y}.

- search space is {x, y}
- subspaces are {x} and {y}